### PR TITLE
Update UI Definitions and Configuration Metadata for LAMP and Wordpress Solutions to work with Dynamic Declarative Views

### PR DESCRIPTION
**Pull request by andrewpham-msft**
In order to work with Dynamic Declarative views, the configuration metadata and UI definition for both the wordpress and LAMP solution have been updated. The UI definition uses patterns that work with the remote Dynamic Dx compiler

It also updates the deployment templates to handle the resourceNamePrefix that was added to the create experiences

### CODE REVIEW

File: solutions/lamp-stack-vm-v1/configuration.metadata.json [hunk 1]
```diff
@@ -140,6 +140,7 @@
                 "tooltip": "The expected maintenance overhead associated with managing this configuration."
             },
             "templateFileName": "azuredeploy-small2mid-noha.json",
+            "uiDefinitionFileName": "azuredeploy-minimal-uidefinition.json",
```
Comment (unrealab) [inline solutions/lamp-stack-vm-v1/configuration.metadata.json:143]: if you're reusing the same form definition, consider removing the "minimal" from the name ? It gives the impression that there are config specific form definitions that we've not linked correctly.

**Comment by unrealab** [comment]
Have you tested that the deployments for all 3 variations work ? These scripts are super sensitive to little changes.

**Review by unrealab — CHANGES_REQUESTED** [review]

File: solutions/lamp-stack-vm-v1/configuration.metadata.json (hunk 1 above)
Comment (andrewpham-msft) [reply solutions/lamp-stack-vm-v1/configuration.metadata.json]: Renamed the form definition for both the LAMP and wordpress solutions to `uiformdefinition.json`

**Comment by andrewpham-msft** [comment]
> Have you tested that the deployments for all 3 variations work ? These scripts are super sensitive to little changes.

Yup! Tested all variations to ensure deployments still work

**Review by unrealab — APPROVED** [review]

**Pull request merged**